elide-async/src/main/java/com/yahoo/elide/async/service/DefaultAsyncQueryDAO.java

```
@@ -114,27 +106,26 @@ public AsyncQuery updateStatus(String asyncQueryId, QueryStatus status) {
                  }
                  return loaded;
              });
-        } catch (ParseException e) {
+        } catch (Exception e) {
```

This is too broad and is swallowing exceptions that need to be propagated.